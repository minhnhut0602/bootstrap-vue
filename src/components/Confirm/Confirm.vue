<template>
    <Modal
        title="Confirm"
        text="You are about to do something that requires confirmation. Do you wish to continue?"
    >
        <template #footer>
            <Btn
                data-bs-dismiss="modal"
                variant="light"
            >
                Cancel
            </Btn>

            <Btn variant="danger">
                Confirm
            </Btn>
        </template>
    </Modal>
</template>

<script lang="ts" setup>
import Btn from '../Button';
import Modal from '../Modal.vue';
</script>
