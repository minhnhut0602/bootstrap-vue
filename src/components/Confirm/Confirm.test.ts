import Confirm from '.';

describe('template', () => {
    componentRenderTest(
        Confirm,
        // Force an id to prevent snapshots from failing because of uuid.
        {props: {id: 'dont-change-please'}},
        false,
    );
});
