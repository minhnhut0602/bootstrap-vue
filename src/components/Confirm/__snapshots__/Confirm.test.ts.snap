// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`template renders default 1`] = `
"<div id=\\"owow-982f85ac-0973-41db-9f76-ba666ac93662\\" class=\\"modal\\">
  <div class=\\"modal-dialog\\">
    <div class=\\"modal-content\\">
      <div class=\\"modal-header\\">
        <h5 class=\\"modal-title\\">Confirm</h5><button class=\\"btn-close\\" data-bs-dismiss=\\"modal\\" type=\\"button\\"></button>
      </div>
      <div class=\\"modal-body\\">You are about to do something that requires confirmation. Do you wish to continue?</div>
      <div class=\\"modal-footer\\"><button class=\\"btn btn-md btn-light\\" data-bs-dismiss=\\"modal\\"> Cancel </button><button class=\\"btn btn-md btn-danger\\"> Confirm </button></div>
    </div>
  </div>
</div>"
`;
